mysql/datadog_checks/mysql/data/conf.yaml.example

```diff
@@ -1,16 +1,24 @@
+## All options defined here are available to all instances.
+#
 init_config:
-    ## @param global_custom_queries - list - optional
-    ## See `custom_queries` defined below.
-    ##
+
+    ## @param global_custom_queries - mapping - optional
```

Comment:
Any reason to use mapping? It's a list.

Reply:
Looks like my spec types fu still needs work ;-)